step.yml

```diff
@@ -114,32 +114,40 @@ inputs:
       description: |
         If is empty then the output name is "app-release-bitrise-signed".
         Else it's the specified name. Do not add extensione here.
+  - verbose_log: "no"
+    opts:
+      title: "Enable verbose logging?"
+      description: Enable verbose logging?
+      is_required: true
+      value_options:
```

Comment:
Note: I m not sure what we use in other places, but if we tend to use true and false, then I think we should use it here too, to make step.yml-s more coherent. If yes and no used widely, ignore this comment. :)

Reply:
Fixed